include/swift/IDE/Formatting.h

```diff
@@ -17,7 +17,9 @@ namespace swift {
 namespace ide {
 
 struct CodeFormatOptions {
+public:
   bool UseTabs = false;
```

Comment:
I agree. I think this structure will end up evolving into a "style" abstraction that is more scalable.